src/app/cube/usage-stats/heat-map/heat-map.component.scss

```diff
@@ -3,7 +3,7 @@
 
   background: white;
   border-radius: 2px;
-  padding: 20px;
+  padding: 40px;
```

Comment:
Why would the removal of bootstrap required a doubling of the padding here? Could the root cause of this be the removal of a 'box-sizing' rule?

Reply:
So it looks like the container has a padding default because it doesn't fill the entire the viewport.